app/assets/stylesheets/components/_outcome-card.scss

```diff
@@ -0,0 +1,93 @@
+$outcome-card-height: 200px;
+
+.outcome-cards,
+.outcome-card {
+  @include clearfix;
+}
+
+.outcome-card {
+  background-color: $white;
+  color: $base-font-color;
+  display: block;
+  margin-bottom: $base-spacing;
+  padding: $base-spacing;
+  text-align: center;
+  white-space: normal;
+
+  @include media($medium-screen-up) {
+    @include span-columns(6 of 12);
+    @include omega(2n);
+    height: $outcome-card-height;
+    text-align: initial;
+  }
+}
+
+.outcome-card-unmatched {
+  border: $base-border;
+  transition: $base-transition;
+
+  &:focus,
+  &:hover {
+    border-color: $mit-red;
+    color: inherit;
+    transform: translate3d(0, -$smallest-spacing, 0);
```

Comment:
You're right, but I'm using `translate3d` in this case to kick in hardware acceleration for better performance.